kratos/python/add_matrix_to_python.cpp

```diff
@@ -68,6 +64,7 @@ namespace Python
         binder.def("__itruediv__", [](TMatrixType& m1, const typename TMatrixType::value_type& value){ m1/=value; return m1;}, py::is_operator());
 
         binder.def("__str__", PrintObject<TMatrixType>);
+        binder.def("copy", [](const TMatrixType& self){TMatrixType aux(self.size1(), self.size2()); for (std::size_t i = 0; i < self.size1(); ++i) {for (std::size_t j = 0; j < self.size2(); ++j) {aux(i,j) = self(i,j);}}; return aux;} );
```

Comment:
Can you please shift it up by one line such that the __str__ method is the lowest one?

Other than that can you please add a test?

Reply:
Done